test/studies/590o/blerner/nilClassCheck.chpl

```diff
@@ -7,7 +7,7 @@ const D = {1..10};
 var A: [D] C;
 
 for i in 1..5 {
-  A(i) = new C(i);
+  A(i) = new unmanaged C(i);
```

Comment:
they couldn't be borrowed because they'd be deallocated at the end of the loop iteration. I think it's OK for them to be unmanaged in the context of this test.